src/Core/Mesh/TriangleMesh.hpp

```diff
@@ -150,8 +168,10 @@ class TriangleMesh {
         m_normalsHandle = m_vertexAttribs.addAttrib<NormalAttribHandle::value_type>( "in_normal" );
     }
 
-    /// Access the attribute manager
+    /// Access the attribute manager.
     const AttribManager& attribManager() const { return m_vertexAttribs; }
+
+    /// Access the attribute manager.
     AttribManager& attribManager() { return m_vertexAttribs; }
```

Comment:
The `AttributeManager` cannot be visible as we restrict its interface in `TriangleMesh` (in_position, ...).
This method needs to be removed.

Reply:
This method is used by the TopologicalMesh, but since it is a friend class it is useless.
So yes I agree to remove it, @dlyr are you ok with that?

Reply:
It has no sense to keep this function, has it breaks the encapsulation needed to store the positions and normals.
If you keep it, anyone can delete or modify the positions and normals without any control.

Reply:
it has been!